packages/test/stochastic-test-utils/src/types.ts

```diff
@@ -74,6 +74,11 @@ export interface SaveInfo {
 	filepath: string;
 }
 
+export interface SavedInfo {
```

Comment:
Are you expecting that we'll eventually put more fields in this interface? If not, then passing the filepath directly as a string seems sufficient. No need to introduce a new interface for that.